src/pages/Cart.js

```diff
@@ -64,7 +61,9 @@ const Cart = () => {
                   <button className="card-2-button-1">Go to Cart</button>
                 </div>
                 <div>
-                  <button className="card-2-button-2">Remove from Wishlist</button>
```

Comment:
I like how you are refactoring code with all the functionalities. Please take care of extra spaces too.